app/assets/javascripts/two-pane.js

```diff
@@ -28,6 +28,13 @@ $(document).on('click', "a[href$='new'].new_two_pane", function(e) {
   }
 });
 
+$(document).on('click', "a[href$='edit'].edit_two_pane", function(e) {
```

Comment:
I have checked - this is the only page with css class `edit_two_pane`. I can couple it more closely if you'd - or add it as a separate pr